### DISCUSSION TITLE
Change map size?

### DISCUSSION THREAD

**Discussion by Tigwin** [teslamate-org/teslamate#2725] [Ideas]
Hello, I have a 4k monitor and I'd like to see my entire state with some level of detail but the visited?orgId=1 page isn't resizable, so I have to pan around and its not the best. Any way to make it larger?

**Comment by tobiasehlert** [comment]
@Tigwin, are you talking about the Grafana _**Visited**_ dashboards or some other location?

**Reply by Tigwin** [reply]
yes, "visited". I gave the url in my post above. 

The map scales width but not height. I know height isn't as easy to scale, just letting us specify a height would be cool

**Reply by tobiasehlert** [reply]
Alright, I wanted to know what view you are referring to before answering.
And no, there is only one map-dashboard in Grafana. (You could have meant the Leaflet JS map in TeslaMate page as well, so therefore I wanted to ask.)

As of what I am aware, it's not possible to set a dynamic height in Grafana on a panel. Setting a width is much "easier" to do in web compared to scaling in height, which is supposed to look correct too.

Anyhow, you can adjust the dashboard and save the changed dashboard, but there is a risk that your change will be lost in an upgrade of Grafana later on. In this case you could always do a copy of the dashboard and update the copy and then an upgrade of TeslaMate should not destroy your local changes.
